### ISSUE TITLE
macOS: `scrolling.smooth true` makes `scroll down/up` not work

### ISSUE THREAD

**Issue by Timothee** [qutebrowser/qutebrowser#6840]
**Version info**:
```
qutebrowser v2.4.0
Git commit: 36ffff2f6 on HEAD (2021-10-21 18:50:06 +0200)
Backend: QtWebEngine 5.15.2, based on Chromium 83.0.4103.122
Qt: 5.15.2
```
macOS Monterey v12.0.1

**Does the bug happen if you start with `--temp-basedir`?**:

[I need to try, will try and update after I finish this bug report.]

**Description**
After upgrading to macOS Monterey, `j` and `k` (i.e. `scroll down` and `scroll up`) stopped working. I isolated the issue to having `scroll.smooth` set to `True`.

**How to reproduce**
* open a page long enough to require scrolling on qutebrowser
* press `j`

**Expected**: it scrolls down
**Actual**: it does nothing.

If I run the command `:set scrolling.smooth false`, it starts working. If I run `set scrolling.smooth true`, it stops again.

**Comment by The-Compiler** [comment]
so I assume this is specific to Monterey and worked fine before, right?

**Comment by Timothee** [comment]
Yeah, it worked fine before and stopped working after the upgrade.

**Comment by angelus2014** [comment]
I can confirm this issue, just upgraded to Monterey and it stopped working. Changing 'scrolling.smooth' to 'false' seems to be the only workaround.

**Comment by The-Compiler** [comment]
Also see #6085 FWIW - something about smooth scrolling must be rather special on macOS...

**Comment by The-Compiler** [comment]
By the way, if anyone on Monterey could try to reproduce #6833 and add a comment there, that'd be great.

**Comment by angelus2014** [comment]
> Also see #6085 FWIW - something about smooth scrolling must be rather special on macOS...

Well, since release 12 (Monterey) apparently. Before I didn't have issues.


---
With regards,

Angelo Machils

**Comment by zmre** [comment]
I also just ran into this. Couldn't figure out why I couldn't scroll with j/k. Turning off smooth scrolling fixes the problem, but leaves things jerky.

**Comment by mamsterdam11** [comment]
Commenting since I also am running into the same issue on MacOS Monterey

**Comment by jakestrahm** [comment]
Same issue.

**Comment by The-Compiler** [comment]
PSA, the "I have the same issue" comments do nothing but add noise. If you have nothing substantial to add, please consider [simply upvoting the issue](https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) instead - thanks!
